ui/views_test.py

```diff
@@ -374,26 +374,6 @@ def test_program_thumbnail(self):
         resp = self.client.get('/')
         self.assertContains(resp, image.get_rendition('fill-690x530').url)
 
-    def test_courses_enrollment_text(self):
```

Comment:
I mostly just care that we are rendering the courses in the correct order (using `position_in_program`). Can you write a test for that if one doesn't exist?